dash.el

```diff
@@ -354,7 +356,7 @@ See also `--filter'."
   `(--filter (not ,form) ,list))
 
 (defun -remove (pred list)
-  "Return a new list of the items in LIST for which PRED returns nil.
+  "Return a new list of the items in LIST for which PRED return nil.
```

Comment:
This is not proper english, unfortunatelly the check is not smart enough to see that.  Only the first "return" should be imperative.